playground/nextjs-app-router/components/form/checkout-options.tsx

```diff
@@ -23,78 +23,78 @@ export function CheckoutOptions() {
     chargeId: '',
   });
 
-  return (
-    activeComponent === OnchainKitComponent.Checkout && (
-      <div className="grid gap-2">
-        <Label htmlFor="chain">Checkout Types</Label>
-        <RadioGroup
-          id="pay-type"
-          value={checkoutTypes}
-          className="flex items-center justify-between"
-          onValueChange={(value) => setCheckoutTypes?.(value as CheckoutTypes)}
-        >
-          <div className="flex items-center gap-2">
-            <Label
-              htmlFor="pay-type-product-id"
-              className="flex cursor-pointer items-center gap-2 rounded-md border p-2 [&:has(:checked)]:bg-muted"
-            >
-              <RadioGroupItem
-                id="pay-type-product-id"
-                value={CheckoutTypes.ProductID}
-              />
-              Product ID
-            </Label>
-            <Label
-              htmlFor="pay-type-charge-id"
-              className="flex cursor-pointer items-center gap-2 rounded-md border p-2 [&:has(:checked)]:bg-muted"
-            >
-              <RadioGroupItem
-                id="pay-type-charge-id"
-                value={CheckoutTypes.ChargeID}
-              />
-              Charge ID
-            </Label>
-          </div>
-        </RadioGroup>
-
-        {checkoutTypes === CheckoutTypes.ChargeID && (
-          <div>
-            <Label htmlFor="charge-id">Charge ID</Label>
-            <Input
-              id="charge-id"
-              value={productOptions.chargeId}
-              placeholder="Enter Charge ID"
-              onChange={(e) => {
-                setProductOptions({
-                  ...productOptions,
-                  chargeId: e.target.value,
-                });
-                setCheckoutOptions?.({
-                  ...checkoutOptions,
-                  chargeId: e.target.value,
-                });
-              }}
+  return activeComponent === OnchainKitComponent.Checkout ? (
```

Comment:
This is required so the component doesn't return `false` in any case.

Reply:
if we have to have these checks for each component specific option is there any reason not to just include them all?

Reply:
Yeah i'm actually not sure why this was included in the first place